tests/datatypes/test_str.py

```diff
@@ -276,6 +276,39 @@ def test_len(self):
             print(len(s))
         """)
 
+    def test_center(self):
+        # test ok
+        self.assertCodeExecution("""
+            s = "abc"
+            print(s.center(10, "-"))
+            """)
+        # less length
+        self.assertCodeExecution("""
+            s = "abc"
+            print(s.center(2,"-"))
+            """)
+        # more than one fillChar
+        self.assertCodeExecution("""
+            s = "abc"
+            try:
+                print(s.center(10,"-*"))
+            except TypeError as err:
+                print(err)
+            """)
+        # called method is not string
+        self.assertCodeExecution("""
+            s = 123
+            try:
+                print(s.center(10,"-"))
+            except AttributeError as err:
+                print(err)
+            """)
+        # bool as width
+        self.assertCodeExecution("""
+            s = "abc"
+            print(s.center(True,"-"))
+            """)
```

Comment:
so there should be only one assertCodeExecution call in the test_center function and I will write test in a single multiline string. did i get it right?